dbcon/joblist/tuple-bps.cpp

```diff
@@ -1582,11 +1585,19 @@ bool TupleBPS::processSingleFilterString(int8_t BOP, int8_t colWidth, int64_t va
                 filterString += 8;
                 break;
 
+            case 16:
+                bigVal2 = *((int128_t*) filterString);
+                filterString += 16;
+                break;
+
             default:
                 throw logic_error("invalid column width");
         }
 
-        thisPredicate = compareSingleValue(COP, val, val2);
+        if (colWidth < 16)
```

Comment:
I think we should replace this with a constant from datatypes::

Reply:
We should reduce a number of hardcoded values around the codebase I believe.

Reply:
Agreed. Changing this.